test/fixtures/conf-invalid-local/local.ts

```diff
@@ -1,6 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const conf = {
 	other: 'yes',
+	// eslint-disable-next-line
 	regexp: new RegExp('tes(t-2'),
```

Comment:
missing explanation
or maybe a fix ? 

```suggestion
	regexp: /tes(t-2/,
```